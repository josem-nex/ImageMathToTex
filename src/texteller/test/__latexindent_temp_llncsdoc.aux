\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand*\HyPL@Entry[1]{}
\HyPL@Entry{0<</S/D>>}
\@writefile{toc}{\contentsline {section}{\numberline {1}Introduction}{3}{section.0.1}\protected@file@percent }
\@writefile{toc}{\contentsline {section}{\numberline {2}How the data was obtained}{3}{section.0.2}\protected@file@percent }
\@writefile{toc}{\contentsline {section}{\numberline {3}Methodology}{3}{section.0.3}\protected@file@percent }
\@writefile{toc}{\contentsline {section}{\numberline {4}Implementation}{4}{section.0.4}\protected@file@percent }
\@writefile{toc}{\contentsline {subsection}{\numberline {4.1}Tokenization}{4}{subsection.0.4.1}\protected@file@percent }
\@writefile{toc}{\contentsline {section}{\numberline {5}Result and Transparence}{4}{section.0.5}\protected@file@percent }
\@writefile{toc}{\contentsline {section}{\numberline {6}Improvements}{4}{section.0.6}\protected@file@percent }
\gdef \@abspage@last{5}
