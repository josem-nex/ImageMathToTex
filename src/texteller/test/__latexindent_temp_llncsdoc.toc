\contentsline {section}{\numberline {1}Introduction}{3}{section.0.1}%
\contentsline {section}{\numberline {2}How the data was obtained}{3}{section.0.2}%
\contentsline {section}{\numberline {3}Methodology}{3}{section.0.3}%
\contentsline {section}{\numberline {4}Implementation}{4}{section.0.4}%
\contentsline {subsection}{\numberline {4.1}Tokenization}{4}{subsection.0.4.1}%
\contentsline {section}{\numberline {5}Result and Transparence}{4}{section.0.5}%
\contentsline {section}{\numberline {6}Improvements}{4}{section.0.6}%
