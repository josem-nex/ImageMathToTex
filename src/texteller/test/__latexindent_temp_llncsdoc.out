\BOOKMARK [1][-]{section.0.1}{\376\377\000I\000n\000t\000r\000o\000d\000u\000c\000t\000i\000o\000n}{}% 1
\BOOKMARK [1][-]{section.0.2}{\376\377\000H\000o\000w\000\040\000t\000h\000e\000\040\000d\000a\000t\000a\000\040\000w\000a\000s\000\040\000o\000b\000t\000a\000i\000n\000e\000d}{}% 2
\BOOKMARK [1][-]{section.0.3}{\376\377\000M\000e\000t\000h\000o\000d\000o\000l\000o\000g\000y}{}% 3
\BOOKMARK [1][-]{section.0.4}{\376\377\000I\000m\000p\000l\000e\000m\000e\000n\000t\000a\000t\000i\000o\000n}{}% 4
\BOOKMARK [2][-]{subsection.0.4.1}{\376\377\000T\000o\000k\000e\000n\000i\000z\000a\000t\000i\000o\000n}{section.0.4}% 5
\BOOKMARK [1][-]{section.0.5}{\376\377\000R\000e\000s\000u\000l\000t\000\040\000a\000n\000d\000\040\000T\000r\000a\000n\000s\000p\000a\000r\000e\000n\000c\000e}{}% 6
\BOOKMARK [1][-]{section.0.6}{\376\377\000I\000m\000p\000r\000o\000v\000e\000m\000e\000n\000t\000s}{}% 7
